### ISSUE TITLE
Error on Apply Changes (various mods) once installed

### ISSUE THREAD

**Issue by drew-id** [Vilsol/ficsit-cli#3]
Replicated this on several mods. ie. Refined Power (with Dependencies) and also Better Bladerunners.

Upon applying changes randomly errors on one of the mods.
Reference error in the installation.go file is line 363 or 376 (attached error screenshot)

**Line 363/376:** case updates <- update:

Even if this succeeds, the server is unable to locate the mod files (although i can see them in the Mods directory)

![test](https://user-images.githubusercontent.com/2094110/174425335-715d0278-f648-492c-9053-840014090f6c.png)

Happy to do further testing if required. Please advise.

**Comment by Vilsol** [comment]
@drew-id thanks for the report! Could you please test out the release v0.0.27 and see if you still have the issue? https://github.com/Vilsol/ficsit-cli/releases/tag/v0.0.27

**Comment by drew-id** [comment]
@Vilsol I've ran a test on 0.27 and this appears to work as expected when installing the mods.

![Test_01](https://user-images.githubusercontent.com/2094110/174559723-0cfb6ace-ef8a-4672-b749-9793e0953ee5.png)

However I did notice when starting the server we got this error - it doesn't seem to detect the SML installation. Wasn't sure if it's missing something to troubleshoot further but progress. :)
**Tested using Hub Terminals which was verified to be working on the Client in U6**

![Test_02](https://user-images.githubusercontent.com/2094110/174559770-ea738f3c-84ec-43e3-a71a-a31aef483536.png)

Not sure if you are able to pinpoint why SML isn't being detected, but something to be aware of regardless.
